src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php

```diff
@@ -91,14 +91,6 @@ public function findOneForPayment($id);
      *
      * @return PagerfantaInterface
      */
-    public function createFilterPaginator(array $criteria = null, array $sorting = null);
-
-    /**
-     * @param array $criteria
-     * @param array $sorting
-     *
-     * @return PagerfantaInterface
-     */
     public function createCheckoutsPaginator(array $criteria = null, array $sorting = null);
```

Comment:
How about other `...Paginator` methods? Are they used anywhere? If not we should remove them also.